### DISCUSSION TITLE
Release VsCodium as a COPR repo

### DISCUSSION THREAD

**Discussion by spazziale** [VSCodium/vscodium#1311] [General]
**Describe the bug**
The rpm repo on gitlab is sometimes offline and it slow down dnf by a lot. Releasing VsCodium as a COPR repo should fix it.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**Desktop (please complete the following information):**
 - OS: Fedora 37 Beta
 - Architecture x64
 - Version: latest 
 - App Manager: dnf

**Comment by Bleuzen** [comment]
Maybe even consider getting this on [RPM Fusion](https://rpmfusion.org/) or the official repos.
VSCod(e/ium) is a well known and widely used tool by now, so it would be a good fit to make it easily accessible without adding yet another 3rd party repo.

**Reply by spazziale** [reply]
True

**Reply by beedell-roke** [reply]
@Bleuzen, that's difficult, per [`discussion.fedoraproject.org/t/96211/14`](https://discussion.fedoraproject.org/t/why-is-there-no-official-vs-codium-package/96211/14).

**Reply by daiyam** [reply]
Thanks for pointing the thread.

I don't think VSCodium should rely on shared Electron/NodeJS. It would be different 20 years ago with hdd constraints but nowadays, it isn't an issue.

**Reply by beedell-roke** [reply]
@daiyam, space is, indeed, a very unimportant factor for most of *my* devices. Security and crash reporting are more so. [^1] However, I also own a Fairphone 4 and LINX 1010, which are both very slow (so no Flatpak OSTree is a benefit), and the latter of which is space-constrained.

[^1]: [`abrt/abrt/issues/1196`](https://github.com/abrt/abrt/issues/1196#issue-186523991)

**Reply by daiyam** [reply]
VSCodium/VSCode is a memory-hungry IDE. On my system, 1.5Go for only 1 window and 7 files opened...

**Reply by beedell-roke** [reply]
@daiyam, yes, although it's usable when one has a sufficient swap partition enabled, even if small. Regardless, I was referring to CPU cycles and permanent storage, rather than SDRAM.

**Comment by spazziale** [comment]
Any update?

**Comment by reisaraujo-miguel** [comment]
Any update?

**Comment by ghost** [comment]
Any update?

**Comment by segmentationfox** [comment]
Any update?
